theme/layouts/_breadcrumbs.ejs

```diff
@@ -0,0 +1,21 @@
+<%
+const query = site.queryPages;
+if (page.pathname !== '') {
+  const [currentPage] = query(({pathname}) => page.pathname === pathname);
+  const pathnameParts = currentPage.originalPathname.split('/'); %>
+  
+  <nav aria-label='Breadcrumb' id='breadcrumb'>
+  <ol>
+    <% 
+    for (let i=0; i < pathnameParts.length; i++) {
+      const pickOriginalPathname = pathnameParts.slice(0, i).join("/");
+      const [{pathname, title}] = query(({originalPathname}) => 
+        originalPathname == pickOriginalPathname); %>
+        <li class='parts'>
```

Comment:
Suggestion: We don't need to define classname to style this element, instead you can also do `#breadcrumb li:not([aria-current='page'] )`? Which seem to be more descriptive rather than `.parts` and provide cleaner Markup.